### DISCUSSION TITLE
wrong course

### DISCUSSION THREAD

**Discussion by kilian2402** [ls1intum/Artemis#9513] [Q&A] [answered]
How can I drop a course in Artemis? I accidentally registered for the wrong course which sounded similar to the one I actually need. Do I need to contact the responsible Prof for this or can I revoke this on my own?

**Comment by MaximilianAnzinger** [comment, marked as answer]
Hi @kilian2402 ,

thank you for reaching out. If you professor has enabled the unenrollment you can do so on your own. Otherwise you'll have to contact your lecturer. Most likely multiple students are affected, thus this might be a good opportunity to tell your instructor that students could do this on their own 😉
